nameko/containers.py

```diff
@@ -421,6 +430,9 @@ def _run_worker(self, worker_ctx, handle_result):
                 _log.debug('handling result for %s', worker_ctx)
 
                 with _log_time('handled result for %s', worker_ctx):
+                    # TODO: retry handle_result once, catch any exception
```

Comment:
Hmm, yes. I think we can scratch this.

I wanted to apply some kind of retry policy for replies, so that (for example) an intermittent connection error would be overcome. That is clearly the responsibility of the entrypoint though.